source/server/LocalServer.ts

```diff
@@ -164,6 +164,10 @@ module Print.Server {
 								child_process.spawn("nautilus", ["--browser", path]).on("error", (error: any) => {
 									console.log("Failed to spawn gnome-terminal. " + error)
 								});
+							else if(urlPathList[3] == "android")
+								child_process.exec("tools/android/flash_vidhance.sh", { cwd: (path + "/ooc-vidproc") }, console.log).on("error", (error: any) => {
```

Comment:
I could never get it to work then. Probably because of my lack of typescript skillz, but this seems to work and the script is quick enough. :)